jdk_17_maven/em/embedded/rest/familie-ba-sak/target/classes/logback.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
files under `target` should never be added to Git

Reply:
I missed to add it to the `.gitignore`. Will remove it.